scripts/scripts.js

```diff
@@ -165,7 +190,11 @@ export function decorateMain(main) {
  * @param {Element} doc The container element
  */
 async function loadEager(doc) {
-  document.documentElement.lang = 'en';
+  // Default to en
+  const lang = window.location.pathname.split('/').indexOf('jp') !== -1 ? 'jp' : 'en';
+  document.documentElement.lang = lang;
+  await fetchPlaceholders(`/prisma/prisma-cloud/${lang}`);
```

Comment:
I think we should fetch placeholders lazily and replace text as needed - we could make a `getPlaceholders()` export from scripts.js that wraps `fetchPlaceholders`, and uses the lang to avoid this template string everywhere we need placeholders though - wdyt?

Reply:
I created https://github.com/hlxsites/prisma-cloud-docs-website/issues/6 to follow up.